persistence/src/main/resources/changelog/changelog.xml

```diff
@@ -699,6 +699,12 @@
       <column name="TIMESTAMP"/>
     </createIndex>
 
+    <createIndex indexName="IDX_ARENDE_PATIENT_PERSON_ID_TIMESTAMP"
```

Comment:
You cannot change previous change-sets... Instead you have to create a new change-set where you create this index.